packages/expect/src/types.ts

```diff
@@ -81,6 +81,7 @@ export type Expect = {
   objectContaining(sample: Record<string, unknown>): AsymmetricMatcher;
   stringContaining(expected: string): AsymmetricMatcher;
   stringMatching(expected: string | RegExp): AsymmetricMatcher;
+  withinDeadline<T>(promise: Promise<T>): Promise<T>;
```

Comment:
should have an implementation in expect that throws and says the runner must implement this function

Reply:
I dumped one in `index.ts`. I was copying `addSnapshotSerializer` (see just above with the TODO), which doesn't do this, but does have an implementation which returns `void 0`.